### PR TITLE
Add filter function to `match: "content"`

### PR DESCRIPTION
**Pull request by MierenManz**
## About

Add filter function for match type content

## Status

- [x] These changes have been tested against Discord API or contain no API change.

### CODE REVIEW

**Comment by MierenManz** [comment]
have fun @Helloyunho 😉

File: src/utils/command.ts [hunk 1]
```diff
@@ -11,29 +11,41 @@ const mentionToRegex: MentionToRegex = {
...
-
-export interface Args<T = unknown> {
+interface ArgsBase {
   name: string
-  match: CommandArgumentMatchTypes
-  defaultValue?: T
-  flag?: string
 }
 
+interface Flag extends ArgsBase {
```
Comment (DjDeveloperr) [inline src/utils/command.ts:18]: Names like `Flag`, `Mention`, etc are too short and not so specific about what they are. Append `Arg` to them? Also, export them for docs.

**Review by DjDeveloperr — CHANGES_REQUESTED** [review]

**Review by Helloyunho — COMMENTED** [review]
Waiting for fixing dj's review. Other than that it looks good to me!

File: src/utils/command.ts (hunk 1 above)
Comment (MierenManz) [reply src/utils/command.ts:18]: I wasn't sure if these needed to be exported because they are components of the type `Args` but I can export those

**Review by Helloyunho — APPROVED** [review]
LGTM!

**Review by DjDeveloperr — APPROVED** [review]
LGTM

**Pull request merged**